### PR TITLE
Use select component in main donation form

### PR DESCRIPTION
**Pull request by marekrozmus**
Little change in main donation form to use select component.

### CODE REVIEW

**Review by RedFox0x20 — APPROVED** [review]
Works nicely to move `CONST` data out of the main code body and to somewhere more easily accessible and changed. Always a bonus.

A nice abstraction layer with `components/form/SelectFormControl.js`, simplifies the process well.

**Review by audreyfeldroy — APPROVED** [review]
All looks great @marekrozmus. @pydanny given your form knowledge it would be great to have you review this too.

File: components/form/SelectFormControl.js [hunk 1]
```diff
@@ -6,18 +6,20 @@ function SelectFormControl({ id, label, options, value, onChange }) {
...
       <select
         className="select select-bordered w-full max-w-xs select-sm"
         id={id}
         name={id}
         onChange={handleChange}
         value={value}
       >
-        {options.map(({ id: optionId, label: optionLabel }) => (
-          <option key={optionId} value={optionId}>
+        {options.map(({ disabled, label: optionLabel, value: optionValue }) => (
```
Comment (pydanny) [inline components/form/SelectFormControl.js:21]: The auto-expanding nature of variables in modern JavaScript is such a nice feature. 😄

**Review by pydanny — APPROVED** [review]
A good change.

**Comment by audreyfeldroy** [comment]
This is wonderful @marekrozmus. Thanks for your help reviewing this @RedFox0x20 and @pydanny. Merged!

**Pull request merged**